src/graph/executor/test/DedupTest.cpp

```diff
@@ -18,7 +18,7 @@ class DedupTest : public QueryTestBase {
   void SetUp() override { QueryTestBase::SetUp(); }
 };
 
-#define DEDUP_RESUTL_CHECK(inputName, outputName, sentence, expected)                   \
```

Comment:
I'm not certain that `res`+`utl` isn't a thing.

Reply:
Just `result`